### PR TITLE
Only run path patcher on Python 2.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
It relies on some builtins that are not available in Python 3, and it's
not clear that it's necessary. Similar functionality will be added if it
turns out we still need something like this after staging Python 3 on
Windows.

### CODE REVIEW

File: src/python/system/path_patcher.py [hunk 1]
```diff
@@ -139,6 +139,11 @@ def patch():
   if not _is_windows():
     return
 
+  # TODO(mbarbella): Verify that this is no longer necessary in Python 3 and
```
Comment (inferno-chromium) [inline src/python/system/path_patcher.py]: Yes it seems not needed, see https://docs.python.org/3/using/windows.html#removing-the-max-path-limitation, can rephrase this TODO.

Comment (mbarbella-chromium) [reply src/python/system/path_patcher.py]: Done!
